node/connection_base.js

```diff
@@ -86,7 +86,13 @@ function connBaseRequest(options) {
     var req = self.buildOutRequest(options);
     self.ops.addOutReq(req);
     req.peer.invalidateScore();
+    req.errorEvent.on(onReqError);
+
     return req;
+
+    function onReqError(err) {
+        self.ops.popOutReq(req.id, req);
```

Comment:
This bug can only be triggered with this PR's, because in this PR, we emit an error in TChannelOutRequest.prototype.sendCallRequestFrame at State.Initial